src/ids-host.ts

```diff
@@ -17,7 +17,7 @@ const ALLOWED_SUFFIXES = [
 const DEFAULT_IDS_HOST = "https://install.determinate.systems";
 const LOOKUP = process.env["IDS_LOOKUP"] ?? DEFAULT_LOOKUP;
 
-const DEFAULT_TIMEOUT = 30_000; // 30 seconds in milliseconds
+const DEFAULT_TIMEOUT = 5; // 5ms
```

Comment:
oh?

Reply:
Forcing the error conditions to take place to make sure it is doing the right thing

Reply:
Ah, gotcha. I was not have expecting the comment to change in that case :sweat_smile: